app/controllers/auctions_controller.rb

```diff
@@ -1,13 +1,13 @@
 class AuctionsController < ApplicationController
   def index
     collection = AuctionQuery.new.public_index
-    @view_model = AuctionsIndexViewModel.new(current_user, collection)
+    @auctions = AuctionsIndexViewModel.new(auctions: collection, current_user: current_user)
```

Comment:
I'm tempted to name the parameter in the model just `user` and we pass it `current_user` from the controller